lib/puppet/provider/base_dsc/powershell.rb

```diff
@@ -1,5 +1,6 @@
 require 'pathname'
 require 'json'
+require_relative '../../../puppet_x/puppetlabs/powershell_manager'
```

Comment:
The metadata.json requires >= 3.5.0 for puppet and >= 3.3.0 for PE so I think we prevent that. Is there somewhere else to check?